app/controllers/api/scores_controller.rb

```diff
@@ -5,7 +5,7 @@ class ScoresController < ApplicationController
     before_action :validate_score_user_id, only: :destroy
 
     def user_feed
-      scores = Score.all.order(played_at: :desc, id: :desc)
```

Comment:
`.includes(:user)` should not be present here, since it was solved in another ticket.